### ISSUE TITLE
Support markup in the Abstract

### ISSUE THREAD

**Issue by zygoloid** [tabatkins/bikeshed#1181]
When the Abstract of a paper is describing syntactic elements of a language, it would be reasonable to want to use markup within the Abstract when naming those elements. However, Markdown appears to escape such markup rather than interpreting it.

Example:

```
Abstract: This paper proposes a new `<=>` operator that determines whether one operand is less than, equal, or greater than another.
Markup Shorthands: markdown yes
```

Here, the \` is ignored and the `<` is HTML-escaped in the output.

**Comment by tabatkins** [comment]
...huh, I'm not sure why it's not working. The markdown-code things are handled separately from the stuff; the other stuff definitely works (I use property autolinks all the time in Abstract).  And a quick look at the code *looks like* markdown-code things should work fine.

I'll look at this in more detail.

**Comment by jfbastien** [comment]
It would be great to have it work in `Title:` as well, though for that it should strip markup in the HMTL title but keep it for the body title (or maybe these need separate meta names? `[TITLE]` and `[STRIPTITLE]` or w/e).

**Comment by tabatkins** [comment]
Note that there is `Title` and `H1`, if you need to distinguish those two because of markup.

**Comment by jfbastien** [comment]
Ah interesting... Doesn't really do what I want though: the boiler plate has `<title>[SHORTNAME]R[LEVEL]: [TITLE]</title>` and `<h1 id="title" class="p-name no-ref">[SHORTNAME]R[LEVEL]<br>[TITLE]</h1>` so it would need repetition despite [what the docs say](https://tabatkins.github.io/bikeshed/#metadata-title). It would be kinda OK if there were an `H1` metadata entry which the boilerplate then used as `<h1 id="title" class="p-name no-ref">[SHORTNAME]R[LEVEL]<br>[H1]</h1>`.

**Comment by Quuxplusone** [comment]
Yes, I just noticed this as well. The current docs even [say](https://tabatkins.github.io/bikeshed/#metadata-custom-warning-text):

> Custom Warning Text specifies the body of a "custom" warning. Like Abstract, it accepts multiple lines, and is parsed as Markdown (as if you’d written them directly into the document).

But Abstract *doesn't* parse its contents as Markdown!

(I thought maybe it might depend on the relative order of the `Abstract:` and `Markup Shorthands: markdown yes` lines in the file, but no, swapping those lines doesn't help.)
